src/torchcodec/encoders/_video_encoder.py

```diff
@@ -51,18 +51,23 @@ def to_file(
             codec (str, optional): The codec to use for encoding (e.g., "libx264",
                 "h264"). If not specified, the default codec
                 for the container format will be used.
+                See :ref:`codec_selection` for details.
             pixel_format (str, optional): The pixel format for encoding (e.g.,
                 "yuv420p", "yuv444p"). If not specified, uses codec's default format.
+                See :ref:`pixel_format` for details.
             crf (int or float, optional): Constant Rate Factor for encoding quality. Lower values
                 mean better quality. Valid range depends on the encoder (commonly 0-51).
                 Defaults to None (which will use encoder's default).
+                See :ref:`crf` for details.
             preset (str or int, optional): Encoder option that controls the tradeoff between
                 encoding speed and compression. Valid values depend on the encoder (commonly
```

Comment:
Here and in the other methods. I suggest this because it wasn't immediately obvious to me what "compression" meant in this case.

```suggestion
                encoding speed and compression (output size). Valid values depend on the encoder (commonly
```